help/import-from-slack.md

```diff
@@ -155,15 +155,15 @@ in mind about the import process:
 - Slack's user roles are mapped to Zulip's [user
   roles](/help/user-roles) in the following way:
 
-| Slack role              | Zulip role    |
-|-------------------------|---------------|
-| Workspace Primary Owner | Owner         |
-| Workspace Owner         | Owner         |
-| Workspace Admin         | Administrator |
-| Member                  | Member        |
-| Single Channel Guest    | Guest         |
-| Multi Channel Guest     | Guest         |
-| Channel creator         | none          |
+       | Slack role              | Zulip role    |
```

Comment:
This should be indented 4 spaces, not 7. (Our buggy parser needs 4, 5, 6, or 7, and a CommonMark parser needs 3, 4, 5, or 6.)

Reply:
Updated to indent 4 spaces.